share/picongpu/examples/Bremsstrahlung/include/picongpu/simulation_defines/param/particle.param

```diff
@@ -36,23 +36,21 @@ namespace particles
      *  unit: none */
     constexpr float_X MIN_WEIGHTING = 1.0;
 
-    constexpr float_X TYPICAL_PARTICLES_PER_CELL = 100;
-
 namespace startPosition
 {
 
-    struct RandomParameter
+    struct RandomParameter100ppc
     {
         /** Count of particles per cell at initial state
          *  unit: none */
```

Comment:
newline `*/`

Reply:
argh, I did not want to touch these just yet since ... ah.